WhiteDotFlat.imageset/Contents.json

```
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "mac",
+      "filename" : “WhiteDotFlat.png",
```

I do not have a mac so I don't know what any of these changes do, but this might be a wrong character?
```suggestion
      "filename" : "WhiteDotFlat.png",
```